blog/_posts/2021-05-18-argocd-private-clusters.md

```diff
@@ -0,0 +1,414 @@
+---
+layout: post
+title: Bring GitOps to your private Kubernetes clusters with ArgoCD and inlets PRO
+description: Learn how to apply GitOps on multiple private Kubernetes cluster with a single Argo CD installation and inlets PRO
```

Comment:
Is the problem that the user wants to apply GitOps, or that they want to deploy applications?

Learn a simple approach to deploying applications across multiple public and private Kubernetes clusters using ArgoCD, inlets PRO and GitOps.